crypto/rand/rand_lib.c

```diff
@@ -231,7 +231,8 @@ static uint64_t get_timer_bits(void)
 # if defined(_POSIX_C_SOURCE) \
      && defined(_POSIX_TIMERS) \
      && _POSIX_C_SOURCE >= 199309L \
```

Comment:
It's like producing least readable code was the purpose. I mean _POSIX_C_SOURCE is about same thing, so why is there another thing in between? Basically both _POSIX_C_SOURCE should be in parentheses to denote it's about same thing. As for _POSIX_TIMERS and MacOS X context. It's defined as -1, while Linux(*) man page says it's positive.

(*) It's not clear if we can hold Linux man pages as official standard, one should check POSIX specs.